Utils/proxy.py

```diff
@@ -94,21 +100,23 @@ def writeToBashrc(proxy, port, username, password, flag):
         filepointer.write('export socks_proxy="{}"\n'.format(make_proxy_url_string(proxy, port, username, password, 'socks')))
         filepointer.close()
 
-def writeDockerServiceConfig(proxy, port, username, password, exceptions=""):
+
+def writeDockerServiceConfig(proxy, port, username, password, exceptions, flag):
     if not os.path.exists(docker_path):
         os.makedirs(docker_path)
 
-    with open(docker, "w") as filepoint:
-        filepoint.write("[Service]\n")
+    filepointer = open(docker_, "w")
+    if not flag:
+        filepointer.write("[Service]\n")
         http_url = make_proxy_url_string(proxy, port, username, password)
         https_url = make_proxy_url_string(proxy, port, username, password, 'https')
         conf_str = 'Environment="HTTP_PROXY={0}" "HTTPS_PROXY={1}"'.format(http_url, https_url)
         if len(exceptions) > 0:
             conf_str += ' "NO_PROXY={}"\n'.format(exceptions)
         else:
             conf_str += '\n'
-
-        filepoint.write(conf_str)
+        filepointer.write(conf_str)
+    filepointer.close()
```

Comment:
This command is not needed due using with command:
https://stackoverflow.com/a/1832589/1573395

Reply:
I understand.
Thank you.
I understood that this should be done.
```
    if not flag:
        with open(docker_, "w") as filepointer:
            filepointer.write("[Service]\n")
            http_url = make_proxy_url_string(proxy, port, username, password)
            https_url = make_proxy_url_string(proxy, port, username, password, 'https')
            conf_str = 'Environment="HTTP_PROXY={0}" "HTTPS_PROXY={1}"'.format(http_url, https_url)
            if len(exceptions) > 0:
                conf_str += ' "NO_PROXY={}"\n'.format(exceptions)
            else:
                conf_str += '\n'
            filepointer.write(conf_str)
````

Reply:
I would like to point out that the argument flag has been removed for the current script and not for how to use the pointer.
I understand that this file is related to not being a backup target, but I think that this script should also be able to restore this file.